pkg/config/v2/route.go

```diff
@@ -296,11 +296,12 @@ type VirtualHost struct {
 
 // RouterMatch represents the route matching parameters
 type RouterMatch struct {
-	Prefix    string            `json:"prefix,omitempty"`    // Match request's Path with Prefix Comparing
-	Path      string            `json:"path,omitempty"`      // Match request's Path with Exact Comparing
-	Regex     string            `json:"regex,omitempty"`     // Match request's Path with Regex Comparing
-	Headers   []HeaderMatcher   `json:"headers,omitempty"`   // Match request's Headers
-	Variables []VariableMatcher `json:"variables,omitempty"` // Match request's variable
+	Prefix         string                 `json:"prefix,omitempty"`    // Match request's Path with Prefix Comparing
+	Path           string                 `json:"path,omitempty"`      // Match request's Path with Exact Comparing
+	Regex          string                 `json:"regex,omitempty"`     // Match request's Path with Regex Comparing
+	Headers        []HeaderMatcher        `json:"headers,omitempty"`   // Match request's Headers
+	Variables      []VariableMatcher      `json:"variables,omitempty"` // Match request's variable
+	DslExpressions []DslExpressionMatcher `json:"dsl_expressions"`
```

Comment:
It is better to add omitempty attribute for the `dsl_expressions` field.

Reply:
done